app/Authentication/AuthServiceProvider.php

```diff
@@ -59,15 +59,14 @@ function () use ($container): RequestAuthenticatorMiddleware {
                     $container->get(ConsumerReadRepository::class),
                     new ConsumerIsInPermissionGroup((string) $container->get('config')['api_key']['group_id']),
                     $container->get(UserPermissionsServiceProvider::USER_PERMISSIONS_READ_REPOSITORY),
-                    $container->get('config')['authenticate_public_routes'] ?? false
                 );
 
                 // We can not expect the ids of events, places and organizers to be correctly formatted as UUIDs,
                 // because there is no exhaustive documentation about how this is handled in UDB2. Therefore we take a
                 // rather liberal approach and allow all alphanumeric characters and a dash as ids.
                 $authenticator->addPublicRoute('~^/offers/?$~', ['GET']);
                 $authenticator->addPublicRoute('~^/(events?|places?)/?$~', ['GET']);
-                $authenticator->addPublicRoute('~^/(events?|places?)/[\w\-]+/?$~', ['GET']);
+                $authenticator->addPublicRoute('~^/(events?|places?)/[\w\-]+/.*?$~', ['GET'], 'embedContributors');
```

Comment:
It seems that there is a small change to this regex, is that needed?

Reply:
no I have restored it